mardi_importer/mardi_importer/zbmath/ZBMathSource.py

```
@@ -533,11 +533,17 @@ def push(self):
                     else:
                         author_strings = info_dict["author"].split(";")
                     authors = []
+                    author_name_strings = []
                     for a, a_id in zip(author_strings, author_ids):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)